snap/snapcraft.yaml

```diff
@@ -1,16 +1,19 @@
 name: butterfly
 grade: stable
 adopt-info: butterfly
-summary: Powerful, minimalistic, cross-platform, opensource note-taking app.
-description: |
-  Butterfly is a note app where your ideas come first. You can paint, add texts and export them easily on every device.
+icon: app/linux/debian/usr/share/icons/hicolor/256x256/apps/dev.linwood.butterfly.png
 base: core22
 confinement: strict
 compression: lzo
+title: 'Linwood Butterfly' #For future, if changes needed, it can be easily done from here
+issues: https://github.com/LinwoodDev/Butterfly/issues #for metadata
+website: https://github.com/LinwoodDev/Butterfly #for metadata in gnome-software, kde-discover
+license: AGPL-3.0 #for metadata
 architectures:
   - build-on: amd64
-    build-for: amd64
-  
+  #- build-on: arm64 # try to build on arm64, as per https://github.com/flutter/flutter/issues/56992
```

Comment:
We can build this, you can uncomment it, the issue was closed 2021

Reply:
We can't, kindly check my action that I have linked there.

Reply:
Okay we can do it for now.
I don't have an arm, I only saw it was closed 2021